<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjouterDepart</class>
 <widget class="QDialog" name="AjouterDepart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AjouterDepart</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Code du vol:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Compagnie:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Heure:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Embarquement:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Porte:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCode">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>JS1610</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCompagnie">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>WESTJET</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHeure">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>19:00</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditVille">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>MONTREAL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEmbq">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>18:15</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPorte">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>G35</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConfirmerDepart">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>460</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonConfirmerDepart</sender>
   <signal>clicked()</signal>
   <receiver>AjouterDepart</receiver>
   <slot>slotAjoutDepart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotAjoutDepart()</slot>
 </slots>
</ui>
